tests/unit/models/transactions/test_account_delete.py

```diff
@@ -0,0 +1,60 @@
+from unittest import TestCase
+
+from xrpl.models.exceptions import XRPLModelException
+from xrpl.models.transactions import AccountDelete
+
+_ACCOUNT = "r9LqNeG6qHxjeUocjvVki2XR35weJ9mZgQ"
+_DESTINATION = "rf7HPydP4ihkFkSRHWFq34b4SXRc7GvPCR"
+
+
+class TestAccountDelete(TestCase):
+    def test_creds_list_too_long(self):
+        """Test that AccountDelete raises exception when credential_ids exceeds max
+        length."""
+        with self.assertRaises(XRPLModelException) as err:
+            AccountDelete(
+                account=_ACCOUNT,
+                destination=_DESTINATION,
+                credential_ids=["credential_index_" + str(i) for i in range(9)],
```

Comment:
```suggestion
                credential_ids=["credential_index_" + str(i) for i in range(MAX_CREDENTIAL_LENGTH + 1)],
```
(I don't remember what the actual field is called, but I recommend using it here)

Reply:
c27ab41b9dd6194250f49c10abe82438f0882903